item-spec/item-spec.md

```diff
@@ -41,7 +41,7 @@ inherited from GeoJSON.
 | id         | string                                                                     | **REQUIRED.** Provider identifier. As most geospatial assets are already defined by some identification scheme by the data provider it is recommended to simply use that ID. Data providers are advised to include sufficient information to make their IDs globally unique, including things like unique satellite IDs.     |
 | type       | string                                                                     | **REQUIRED.** Type of the GeoJSON Object. MUST be set to `Feature`.                                                                                                                                                                                                                                                          |
 | geometry   | [GeoJSON Geometry Object](https://tools.ietf.org/html/rfc7946#section-3.1) | **REQUIRED.** Defines the full footprint of the asset represented by this item, formatted according to [RFC 7946, section 3.1](https://tools.ietf.org/html/rfc7946). The footprint should be the default GeoJSON geometry, though additional geometries can be included. Specified in Longitude/Latitude based on EPSG:4326. |
-| bbox       | [number]                                                                   | **REQUIRED.** Bounding Box of the asset represented by this item. This is to enable more naive clients to easily index and search geospatially. Most software can easily generate them for footprints. Specified in Longitude/Latitude based on EPSG:4326.                                                                   |
+| bbox       | [number]                                                                   | **REQUIRED.** Bounding Box of the asset represented by this item. Specified in Longitude/Latitude based on EPSG:4326 - first two numbers are longitude and latitude of lower left corner, followed by longitude and latitude of upper left corner. This field enables more naive clients to easily index and search geospatially. Most software can easily generate them for footprints. STAC compliant APIs are required to compute intersection operations with the item's geometry field, not it's bbox.                 |
 | properties | Properties Object                                                          | **REQUIRED.** A dictionary of additional metadata for the item.                                                                                                                                                                                                                                                                    |
```

Comment:
This is where I saw a reference to the naive implementation that led me to believe that a simple search based on bounding boxes intersection was allowed for STAC APIs implementations. After reading again it is clear that this refers to general clients.
Not sure about the lower left - upper right corner convention, suggesting the same convention defined for the search API.